src/components/OSCALProfile.test.js

```diff
@@ -24,7 +24,7 @@ function testOSCALProfile(parentElementName, renderer) {
   jest.setTimeout(6000);
   test(`${parentElementName} displays controls`, async () => {
     renderer();
-    const result = await screen.findByText("ac-1", {
+    const result = await screen.findByText("AC-1", {
```

Comment:
This changes because we now properly use the _label_ which does specify the value in uppercase.